package.json

```diff
@@ -222,7 +224,8 @@
   },
   "husky": {
     "hooks": {
-      "commit-msg": "commitlint -E HUSKY_GIT_PARAMS"
+      "commit-msg": "commitlint -E HUSKY_GIT_PARAMS",
+      "pre-push": "npm run format"
```

Comment:
At the moment the `npm run format` won't fail if `eslint` or `prettier` modify the files, so the committer will have to do another commit and push it.
Do we want those commands to fail and prevent the push? That will save some CI jobs that are bound to fail.

Reply:
Yes, absolutely, we do. I had wrong assumptions on `eslint` and `prettier` exit codes. I will fix this.